### DISCUSSION TITLE
Is it possible to use Unleash proxy Unleash gem?

### DISCUSSION THREAD

**Discussion by pedrogusfer** [Unleash/unleash#6975] [Q&A] [answered]
I'm seeing in the code that using the gem `unleash` I'm not able to use the unleash proxy. 

Is there a way to use the gem with the proxy?

**Comment by sighphyre** [comment, marked as answer]
Hey @pedroaugustofsilva, this isn't possible. The Ruby gem is specifically for backend clients, the proxy sends a very different set of data
